src/test/java/GitHubTest.java

```diff
@@ -4,7 +4,7 @@ public class GitHubTest extends  BaseTest{
     @Test
     public void smokeIssuesTab() {
```

Comment:
it's a good idea to separate this test into several and a test name should describe the key checking of the test. Your test name has a pretty common name - it's an indicator that it is needed to divide it